src/test/ui/fn_once-moved.stderr

```diff
@@ -1,12 +1,15 @@
 error[E0382]: use of moved value: `debug_dump_dict`
   --> $DIR/fn_once-moved.rs:21:5
    |
+16 |         for (key, value) in dict {
+   |                             ---- dict moved here
```

Comment:
I changed it to `span_note` and it looks like:
```
error[E0382]: use of moved value: `debug_dump_dict`
  --> src/test/ui/fn_once-moved.rs:21:5
   |
20 |     debug_dump_dict();
   |     --------------- value moved here
21 |     debug_dump_dict();
   |     ^^^^^^^^^^^^^^^ value used here after move
   |
   = note: closure cannot be invoked more than once because it moves the variable `dict` out of its environment
note: `dict` moved here
  --> src/test/ui/fn_once-moved.rs:16:29
   |
16 |         for (key, value) in dict {
   |                             ^^^^

error: aborting due to previous error
```
The flow of the error message is indeed better.